governance-proposal/TSC-Charter-Draft.md

```diff
@@ -0,0 +1,103 @@
+* Stability (e.g. presence of test suites, stable APIs and use of an appropriate source-code control system).
+* Predictability of releases
+* Alignment with Node.js Foundation’s goals and priorities.
+
+The TSC and entire technical community will follow any processes as may be specified by the Board relating to the intake and license compliance review of contributions, including the Node.js Foundation IP Policy.
+
+## Section 7. Elections
+
+Leadership roles in Node.js Foundation will be peer elected representatives of the community.
+
+For election of persons (TSC Chairperson, Maintainers, etc.) a multiple-candidate method should be used, e.g.:
+
+* [Condorcet](http://en.wikipedia.org/wiki/Condorcet_method) or
+* [Single Transferable Vote](http://en.wikipedia.org/wiki/Single_transferable_vote)
+
+Multiple-candidate methods may be reduced to simple election by plurality when there are only two candidates for one position to be filled. No election is required if there is only one candidate and no objections to the candidates election. Elections shall be done within the Projects by the Collaborators active in the Project.
+
+Each Core Project’s Collaborators shall elect one Maintainer from the Collaborators on the project to serve on the TSC. There may be only one Maintainer per Core Project that shall be nominated and elected by the Collaborators within the Core Project.
+
+The TSC will elect from amongst voting TSC members a TSC Chairperson to work on building an agenda for TSC meetings and represent the TSC to the Board for a term of one year according to the Node.js Foundation’s By-laws. The TSC shall hold annual elections to select a TSC Chairperson; there are no limits on the number of terms a TSC Chairperson may serve.
+
+## Section 8. Voting
+
+For internal project decisions, Collaborators shall operate under Lazy Consensus. The TSC shall establish appropriate guidelines for implementing Lazy Consensus (e.g. expected notification and review time periods) within the development process.
+
+The TSC follows a [Consensus Seeking](http://en.wikipedia.org/wiki/Consensus-seeking_decision-making) decision making model. When an agenda item has appeared to reach a consensus the moderator will ask "Does anyone object?" as a final call for dissent from the consensus.
+
+If an agenda item cannot reach a consensus a TSC member can call for either a closing vote or a vote to table the issue to the next meeting. The call for a vote must be seconded by a majority of the TSC or else the discussion will continue. Simple majority wins, with the following exceptions, which will require the affirmative vote of two-thirds of the members of the TSC to pass:
+
+* Adding or removing members of the TSC
+* Changes to the TSC Charter (which also require Board approval)
+
+## Section 9. Project Roles
+
+The Node.js Foundation git repository is maintained by the TSC and additional Collaborators who are added by the TSC on an ongoing basis.
```

Comment:
Again, minor question, do we want to specify the technology of 'git' here? Would a more neutral 'repository' be better?